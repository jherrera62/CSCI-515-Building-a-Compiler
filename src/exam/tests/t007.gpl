int a=4444;
double q = a > 5   ? 99 : 88;
double r = a < 5   ? 99.9 : 88.8;
string name = q>r ? "bilbo" : "baggins";
