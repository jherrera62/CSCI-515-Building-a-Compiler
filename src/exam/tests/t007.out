gpl.cpp::main()
  input file(tests/t007.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int a = 4444
string name = "bilbo"
double q = 99
double r = 88.8
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().

User quit program. Printing the symbol table.
int a = 4444
string name = "bilbo"
double q = 99
double r = 88.8
